screens/SendSetupScreen.js

```diff
@@ -240,7 +241,7 @@ const SendSetupScreen = ({
   };
 
   let availableAmount = new BigNumber(0);
-  if (tokenId) {
+  if (tokenId && symbol !== undefined) {
```

Comment:
Why do we need this symbol check here, and throughout the component?  
If it's needed, lets create a variable to track this such as 
`const isToken === token && symbol !== undefined`
and use that throughout.  

I think though we don't want to check for the symbol, as `symbol` is NOT a requirement for SLP tokens, and some do not have one, such as the SLP torch.    I think adding this check will make those tokens unsendable.